guides/plugins/plugins/framework/event/finding-events.md

```diff
@@ -279,3 +279,7 @@ There's a vue browser extension which can greatly help will development in gener
 
 [Vue.js devtools for Firefox](https://addons.mozilla.org/de/firefox/addon/vue-js-devtools/)
 [Vue.js devtools for Google Chrome](https://chrome.google.com/webstore/detail/vuejs-devtools/nhdogjmejiglipccpnnnanhbledajbpd)
+
+## Flow builder events
+
+In Flow Builder, all events data will be stored in the `StorableFlow`, hence the `getAvailableData` function can no more be used to get data from the Flow Builder. For more information on this refer to [Create a new trigger (event)](../../../../../guides/plugins/plugins/framework/flow/add-flow-builder-trigger.md#create-a-new-trigger-event) section of this guide.
```

Comment:
We have Flow Builder events, but  `StorableFlow` only works from 6.5, could you add more info clearly?

eg: From 6.5 ...